src/prompt.rs

```diff
@@ -201,58 +201,41 @@ impl Prompt {
         // needed.
         let safe_prompt = || SAFE_PROMPT.to_string();
 
-        // In-memory cursor using a vector for data.
-        let cursor = Cursor::new(Vec::new());
-
-        let t = term::TerminfoTerminal::new(cursor);
-        if t.is_none() {
-            return safe_prompt();
-        }
-        let mut t = t.unwrap();
+        let bufwtr = BufferWriter::stderr(ColorChoice::Always);
+        let mut buffer = bufwtr.buffer();
+        let mut color = ColorSpec::new();
 
         // Create textual prompt.
-        if t.fg(term::color::GREEN).is_err() {
+        if buffer.set_color(color.set_fg(Some(Color::Green))).is_err() {
             return safe_prompt();
         }
-        if write!(t, "carapace").is_err() {
+        if write!(&mut buffer, "carapace").is_err() {
             println!("Failed to write to term!");
         }
 
         if let Ok(cwd) = env::current_dir() {
-            if t.fg(term::color::BRIGHT_BLUE).is_err() {
+            if buffer.set_color(color.set_fg(Some(Color::Blue))).is_err() {
```

Comment:
I would like to keep the `BRIGHT_BLUE` if possible.

Reply:
Give that a try. I switched to bright blue using an rgb value.